src/app/downloads/downloads-table.component.html

```diff
@@ -3,8 +3,8 @@
 
   <div class="table-navigation">
     <button (click)="toggleExpandedReport()" class="btn-link">
-      <p *ngIf="!expanded">Show more &gt;&gt;</p>
-      <p *ngIf="expanded">Show less &lt;&lt;</p>
+      <p *ngIf="!expanded">Show daily numbers &#187;</p>
```

Comment:
I think you'll want to use the key from the interval here -- the extra info included in the table will only be `daily` if the user has selected Daily interval

Reply:
yep, yep, `routerState.interval.key` (mostly making this comment to say I don't wanna talk about why the interval names are the same as the key, things morph into other things ok. 😺)